_zh-cn/overviews/scala3-book/first-look-at-types.md

```diff
@@ -2,6 +2,7 @@
 title: 类型初探
 type: chapter
 description: This page provides a brief introduction to Scala's built-in data types, including Int, Double, String, Long, Any, AnyRef, Nothing, and Null.
+languages:[en, ru]
```

Comment:
the `languages` field should only go on the English translation, the `zh-cn` pages should have
```yaml
language: zh-cn
```

Reply:
done